cabot/cabotapp/views.py

```diff
@@ -658,7 +659,10 @@ class ServiceCreateView(LoginRequiredMixin, CreateView):
     model = Service
     form_class = ServiceForm
 
-    alert.update_alert_plugins()
+    # this is a hacky place to put this, but it can't run at import time anymore
```

Comment:
I found this approach in the main Cabot repo. [The docs]((https://docs.djangoproject.com/en/1.11/ref/applications/#django.apps.AppConfig.ready).) for the `ready()` method caution against modifying the database because `ready()` gets called with every management command - this means DB operations in `ready()` to run even during tests/dry runs. (This might be acceptable, though?)

*That said*, this might have the same problem - it depends on how urls.py gets parsed...I need to look into this further.

Reply:
Moved this to a `post_migrate` signal, which is what upstream Cabot uses now.